packages/web/config.ts

```diff
@@ -31,4 +31,12 @@ export const CONFIG = {
   googleDataApiKey: process.env.NEXT_PUBLIC_YOUTUBE_API_KEY,
   web3StorageToken: process.env.WEB3_STORAGE_TOKEN || '',
   openseaApiKey: process.env.OPENSEA_API_KEY || '',
+  onboardingGameDataURL:
+    'https://ipfs.fleek.co/ipfs/bafybeig5n6md7kemfjgdbguiepzisrt2y6icxhsj65kyrirs3kqgpv56eu',
+  openseaBaseURL: 'https://opensea.io',
+  polygonscanBaseURL: 'https://polygonscan.com',
+  chievContractAddress: '0x85fCaAFc0dA050FCE685DcB8965F0C1Aa1Ba466b',
+  alchemyRpcUrl: process.env.NEXT_ALCHEMY_ENDPOINT || '',
+  alchemyApiKey:
+    process.env.NEXT_ALCHEMY_API_KEY || 'eoEduI_LCCxfMPe9MKrpvly-qWtBBY7I',
```

Comment:
We are already using an infura id, can we not reuse infura for Cheivs? We don't need to use multiple RPC providers.

Reply:
> We are already using an infura id, can we not reuse infura for Cheivs? We don't need to use multiple RPC providers.

Infura doesn't support Polygon does it? Chieves is on Polygon and I was testing using a ThirdWeb contract on Mumbai, so used my Alchemy account.

Reply:
it does support polygon :)